safe-global/web-core
# fix: navigate to last used Safe when switching chain

## What it solves

Resolves #458

## How this PR fixes it

Switching network via the header (not on Safe creation/loading pages) opens the last used Safe of the chosen network.

## How to test it

Open a Safe on a network (assigning it to 'last used' in the `session` store), switch network and then switch back to the previous on. Observe that the previous Safe automatically opens.

This should not occur in the Safe creation/loading pages.

## Screenshots
![last Safe](https://user-images.githubusercontent.com/20442784/188909783-a35bca5d-2e98-4491-b9dd-2811e9dba371.gif)
